demos/capi/README.md

```diff
@@ -0,0 +1,89 @@
+# C and C++ API inference demo (C/C++)
+
+This demo creates a new docker image openvino/model_server-capi with two example applications that are using c or c++ ovms API.
+
+## Build ovms_shared.so
+To build the capi docker image, you must first build the ovms.tar.gz package with the libovms_shared.so library and ovms.h header. 
```

Comment:
Should we require building from scratch? Shouldnt we promote usage of released ovms.tar.gz package?